data/services/ambari/0.2.2.0/service.xml

```diff
@@ -0,0 +1,27 @@
+<!--
```

Comment:
I think we are moving this to data folder. But we still seem to have the original files in services directory also. Should they be removed? Also, if we are changing the paths, should we change the paths in the code where we refer to them?

Reply:
yes, I planned to remove that as part of next check-in but we can do it now itself. i will delete the top level **service** directory. The references in the code is already modifed